kedro/framework/cli/utils.py

```diff
@@ -401,11 +405,27 @@ def _config_file_callback(ctx, param, value):  # noqa: unused-argument
 
     if value:
         config = OmegaConf.to_container(OmegaConf.load(value))[section]
+        for key, value in config.items():
+            _validate_config_file(key)
         ctx.default_map.update(config)
 
     return value
 
 
+def _validate_config_file(key):
+    """Validate the keys provided in the config file against the accepted keys."""
+    from kedro.framework.cli.project import run
+
+    run_args = [click_arg.name for click_arg in run.params]
+    run_args.remove("config")
+    if key not in run_args:
+        KedroCliError.VERBOSE_EXISTS = False
```

Comment:
@noklam This is a possible "solution" for the `--verbose` messaging, but it's not ideal because it only solves it for when you run --config, but not for other CLI commands that actually don't have a verbose flag.